includes/core.php

```diff
@@ -80,3 +84,73 @@ function maybe_disable_upload_image( $attachment_id ) {
 
 	return $attachment_id;
 }
+
+/**
+ * Adds tweeted status columns for supported post types.
+ */
+function add_tweeted_column() {
+	// Loop through all the supported post types and add tweet status column.
+	$post_types = Utils\get_enabled_post_types();
+	foreach ( (array) $post_types as $post_type ) {
+		add_post_type_support( $post_type, POST_TYPE_SUPPORT_FEATURE );
```

Comment:
this duplicates the functionalities of `set_post_type_supports()`

Reply:
@dinhtungdu I've moved similar code to the other int hook function we already had.